starters/vue3-xstate-css/package.json

```diff
@@ -46,7 +44,6 @@
     "eslint-plugin-vue": "9.3.0",
     "npm-run-all": "4.1.5",
     "prettier": "2.7.1",
-    "start-server-and-test": "1.14.0",
```

Comment:
This package was only used for the e2e tests which aren't set up for this project (only component tests are). Also why I removed the scripts.